Xamarin.Forms.Platform.iOS/Renderers/ActivityIndicatorRenderer.cs

```diff
@@ -12,7 +12,10 @@ protected override void OnElementChanged(ElementChangedEventArgs<ActivityIndicat
 			{
 				if (Control == null)
 				{
-					SetNativeControl(new UIActivityIndicatorView(RectangleF.Empty) { ActivityIndicatorViewStyle = UIActivityIndicatorViewStyle.Gray });
+					if(Forms.IsiOS13OrNewer)
```

Comment:
Just a small change to use an iOS13 API to make sure this is all building correctly against Xcode11/ios13